src/services/audius-backend/Explore.ts

```diff
@@ -121,9 +118,9 @@ class Explore {
     }
   }
 
-  static async getTopFolloweeSaves(limit = 25): Promise<Track[]> {
+  static async getTopFolloweeSaves(limit = 25) {
     try {
-      const tracks = await libs().discoveryProvider.getTopFolloweeSaves(
+      const tracks: UserTrack[] = await libs().discoveryProvider.getTopFolloweeSaves(
```

Comment:
just something for the future, might make sense to start typing libs with a d.ts file